cfme/storage/volume.py

```diff
@@ -191,6 +205,14 @@ def refresh():
         except TimedOutError:
             logger.error('Timed out waiting for Volume to disappear, continuing')
 
+    def edit(self, name):
```

Comment:
**Question:**  Do we have only name editable for volume?

Reply:
Yes, at least as I can see it for volume of Openstack provider.